conf/app.routes

```diff
@@ -114,9 +114,12 @@ DELETE      /application/:applicationId/issue                            control
 
 GET         /schools                                                     controllers.SchoolsController.getSchools(term: String)
 
-POST        /assessor-availability/save/:userId                          controllers.AssessorAvailabilityController.save(userId: String)
-GET         /assessor-availability/find/:userId                          controllers.AssessorAvailabilityController.find(userId: String)
-GET         /assessor-availability/countSubmitted                        controllers.AssessorAvailabilityController.countSubmitted()
+POST        /assessor/save/:userId                                       controllers.AssessorController.saveAssessor(userId: String)
+GET         /assessor/find/:userId                                       controllers.AssessorController.findAssessor(userId: String)
+# TODO: /assessor-availability rename to sth.
```

Comment:
TODOs should only be used for your own purposes and have to be removed before code is merged.